src/network/netdev/batadv.c

```diff
@@ -0,0 +1,94 @@
+/***
+    This file is part of systemd.
+
+    Copyright 2017 Andreas Rammhold <andreas@rammhold.de>
```

Comment:
copy&paste? Please drop the license boilerplate unless absolutely required by your employer or similar.